mako.systemd.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Lightweight Wayland notification daemon
+Documentation=man:mako(1)
+PartOf=graphical-session.target
```

Comment:
Since `graphical-session.target` is not wayland specific I think it is not adapted, mako should be tied to a target (non existant in systemd upstream) like the `wayland-session.target` I ship in my [sway-services](https://github.com/xdbob/sway-services) project.

Reply:
`graphical-session.target` is shipped with upstream systemd, and applies to either Xorg or Wayland[[1]](https://www.freedesktop.org/software/systemd/man/systemd.special.html).

I don't think many people run Xorg with mako installed, and if they do, they're already encountering breakage anyway (it still picks up DBus activation).

`wayland-session.target` is not a standard thing (nor included with systemd). I personally use `sway-session.target`, but this can quickly turn into a bikeshed discussion. I'd rather just keep the existing upstream.